InformationScripting/scripts/sink.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Why is this written in python? Shouldn't we do this in C++? Do we need both, perhaps?